app/src/modules/insights/components/workspace.vue

```diff
@@ -14,7 +14,7 @@
 		>
 			<insights-panel
 				v-for="panel in panels"
-				:key="panel.id"
+				:key="getSimpleHashForPanel(panel)"
```

Comment:
Everything in the panel should already be reactive to the changes made to the props. Instead of refreshing the whole panel component in Vue (and therefore destroying and recreating the contents within), I think the cleaner fix here would be to add that reactivity to the chart component itself whenever the props change 🙂